core/trino-spi/src/test/java/io/trino/spi/predicate/TestRange.java

```diff
@@ -28,29 +28,36 @@
 
 public class TestRange
 {
-    @Test(expectedExceptions = IllegalArgumentException.class)
```

Comment:
There are many more like this across the codebase. I filed https://github.com/trinodb/trino/issues/6974 for them.